mwdb/web/src/components/ShowObject/Actions/PullAction.js

```diff
@@ -1,17 +1,21 @@
 import React, { useContext, useState } from "react";
+import { useHistory, useRouteMatch } from "react-router";
 import { faGlobe } from "@fortawesome/free-solid-svg-icons";
 
 import { APIContext } from "@mwdb-web/commons/api/context";
-import { ConfigContext } from "@mwdb-web/commons/config";
 import { ObjectContext } from "@mwdb-web/commons/context";
 import { ObjectAction } from "@mwdb-web/commons/ui";
 import { ConfirmationModal } from "@mwdb-web/commons/ui";
 
 export default function PullAction() {
     const api = useContext(APIContext);
-    const config = useContext(ConfigContext);
     const context = useContext(ObjectContext);
+    const history = useHistory();
```

Comment:
maybe that would be useRemote function from commons/remotes ?

Reply:
We still need to push so it's required anyway 🤔